charts/tidb-cluster/values.yaml

```diff
@@ -192,6 +193,14 @@ tikv:
   #   effect: "NoSchedule"
   annotations: {}
 
+  ## storeLabels is used to define store label keys
```

Comment:
In order to use the location awareness feature of TiKV, users have to label their k8s nodes with the same labels.